metrics/src/main/java/cucumber/metrics/jmx/PremierMBean.java

```diff
@@ -0,0 +1,13 @@
+package cucumber.metrics.jmx;
+
+public interface PremierMBean {
+
+    public String getNom();
```

Comment:
Cucumber-JVM uses English as a standard for code naming, so probably, IMHO, best to rename to use English words.